samples/third-party/webgl2-half-float-array-converter.js

```diff
@@ -0,0 +1,98 @@
+
```

Comment:
For now, just put this file in the `samples` directory since it isn't third-party yet (sorry if I told you to put it there to begin with).

Also rename it to something short like `HalfFloatUtility.js`.

Likewise, I would rename `HALFFLOAT` below to match (and why is it in all caps?).